ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -843,6 +875,24 @@ public class BottomSheetController: UIViewController {
         return maxHeight
     }
 
+    // Output of `collapsedHeightResolver` wrapped in a cache.
+    private var resolvedCollapsedSheetHeight: CGFloat {
+        let oldContext = lastCollapsedSheetHeightResolutionContext
+        let newContext = ContentHeightResolutionContext(maximumHeight: maxSheetHeight - view.safeAreaInsets.bottom , containerTraitCollection: view.traitCollection)
+
+        if oldContext?.maximumHeight != newContext.maximumHeight || !(oldContext?.containerTraitCollection.containsTraits(in: newContext.containerTraitCollection) ?? false) {
+            lastResolvedCollapsedSheetHeight = collapsedHeightResolver?(newContext) ?? 0
+            lastCollapsedSheetHeightResolutionContext = newContext
```

Comment:
I think this is just my misunderstanding, but when when will `lastCollapsedSheetHeightResolutionContext ` be initialized for the first time? It looks like it would happen here, but in order for it to get here its value would need to be non-nil so that it passes the optional check on line 883.

Reply:
When it's `nil` initially, the equality heck with newContext will fail, so we will go in this branch and set it to newContext. So in other words a nil context is "dirty" and will cause a call to the resolver.